helusers/urls.py

```diff
@@ -16,7 +16,7 @@
 ):
     if not settings.LOGOUT_REDIRECT_URL:
         raise ImproperlyConfigured(
-            "You must configure LOGOUT_REDIRECT_URL to use helusers views."
+            "LOGOUT_REDIRECT_URL setting must be set to use social_auth login/logout."
```

Comment:
Is this the most clear message for a user of this library?

I mean the user adds "helusers" to the project and gets an error that refers "social_auth". Maybe the message could say that this is an error in helusers?

Reply:
It tries to tell that the user is probably trying to use the [Adding tunnistamo authentication](https://github.com/City-of-Helsinki/django-helusers/blob/master/README.md#adding-tunnistamo-authentication) feature from helusers, but didn't follow the instructions in the README fully and is missing a setting. Note that this error isn't raised for every helusers user, only for those that have 

```python
"social_django" in settings.INSTALLED_APPS and "helusers.tunnistamo_oidc.TunnistamoOIDCAuth" in settings.AUTHENTICATION_BACKENDS
```

So that's what it _tries_ to communicate. It's completely possible that this text doesn't fully succeed in that. Any improvement propositions?

Reply:
Something like `...to use social_auth login/logout with helusers.`?

Reply:
Changed the whole message to "LOGOUT_REDIRECT_URL setting must be set to use social_auth login/logout with helusers.". Was that the intention?